pkgs/os-specific/bsd/setup-hook.sh

```diff
@@ -1,3 +1,5 @@
+# shellcheck shell=bash disable=SC2154,SC2164
```

Comment:
https://www.shellcheck.net/wiki/SC2154 -- where's this one trigger?

https://www.shellcheck.net/wiki/SC2164 -- where's this one trigger?

Reply:
Thanks, I'll file that as assumed on future reviews.